packages/core/src/html/linkProcessor.js

```diff
@@ -87,18 +87,34 @@ function convertMdAndMbdExtToHtmlExt(node) {
     if (hasNoConvert) {
       return;
     }
+
     const { href } = node.attribs;
-    const hasMdExtension = href.slice(-3) === '.md';
-    if (hasMdExtension) {
-      const newHref = `${href.substring(0, href.length - 3)}.html`;
-      node.attribs.href = newHref;
+
+    const hasMdExtension = href.includes('.md');
+    const hasMbdExtension = href.includes('.mbd');
+
+    const isExclusiveOr = (!hasMdExtension && hasMbdExtension) || (hasMdExtension && !hasMbdExtension);
+    if (!isExclusiveOr) {
+      // 1) has both '.md' and '.mbd' extension OR 2) has neither of the extensions
```

Comment:
what's case 1 for? 👀

Reply:
I was thinking that if both cases were present, then we would deem it as "invalid". I thought that it would be quite rare to have such a case and I don't think I would expect it to happen. What do you think? :O

Reply:
^ https://github.com/MarkBind/markbind/pull/1493#discussion_r588983611